src/components/chatroom/DateSeparator.tsx

```diff
@@ -0,0 +1,24 @@
+interface DateSeparatorProps {
+  date: string; // "2025년 9월 9일" 형식
+  isFirstMessage?: boolean; // 첫 메시지인지 여부
+}
+
+const DateSeparator = ({ date, isFirstMessage = false }: DateSeparatorProps) => {
+  return (
+    <div className={`flex justify-center items-center ${isFirstMessage ? 'mb-4' : 'mt-8 mb-4'}`}>
```

Comment:
```suggestion
    <div className={`flex justify-center items-center mb-4 ${!isFirstMessage && 'mt-8'}`}>
```
사소하지만 이렇게 하면 공통 스타일 분리할 수 있을 것 같네요!